src/config/settings.js

```diff
@@ -135,4 +146,52 @@ function parseSaveError(err) {
   return friendlierMsg;
 }
 
+// updateTable is a hash of [integer, function(settings) => void].
+//
+// It is currently empty, but if there are any changes across versions,
+// they should be done in a function that modifies the settings arg.  The main use-cases
+// are for renaming property names, correct values that are no longer valid, and removing
+// obsolete entries. The final step merges in current defaults, so we won't need an entry
+// for every version change, as most changes will get picked up from the defaults.
+//
+// For example:
+/*
+let updateTable = {
+  3: function(settings) {
+      // Implement setting change from version 3 to 4
+      if (settings.kubernetes.oldName && !settings.kubernetes.newName) {
+         settings.kubernetes.newName = settings.kubernetes.oldName;
+         delete settings.kubernetes.oldName;
+      }
+      if (settings.kubernetes.bird == "road runner") {
+        settings.kubernetes.bird == "roadrunner" // no spaces wanted
+      }
+   },
+};
+*/
+let updateTable = {
+}
```

Comment:
Silly nit:
```suggestion
};
```

Reply:
done. I think we can configure eslint to fail on missing semi-colons (and whitespace for that matter).

Reply:
yup